sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelData.java

```diff
@@ -49,24 +49,24 @@
     private boolean decommissioned;
 
     /*
-     * The model definition.
+     * The model definition that conforms to the Digital Twins Definition Language (DTDL) v2.
```

Comment:
remove the word "the" before "Digital Twins Definition Language"

Reply:
done